src/theme.ts

```diff
@@ -13,6 +13,8 @@ export const theme = extendTheme({
     global: {
       '*, *:before, *:after': {
         position: 'relative',
+        minHeight: 0,
+        minWidth: 0,
```

Comment:
Might need to reconsider this approach - somebody has mentioned a drawback here: https://twitter.com/chungwu/status/1427413289621876743